### PR TITLE
Update dashboard with correct IP address

### PR DESCRIPTION
**Pull request by bcarnevale**

### CODE REVIEW

**Comment by atl-mk** [comment]
I'm not sure if the PR title is correct, I see other changes. Also the IP address won't help customers.

**Review by atl-mk — APPROVED** [review]

**Comment by Wil312** [comment]
I think we should wait until we've labelled the relevant metrics with a `product` (we'll be removing the hardcoded IPs as these are specific to our environments)

**Review by amcculloch2 — APPROVED** [review]

File: Overview.json [hunk 1]
```diff
@@ -441,18 +455,35 @@
...
       },
       "targets": [
         {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
+          },
           "exemplar": true,
-          "expr": "increase(jvm_gc_collection_seconds_sum{instance=\"localhost:9000\"}[1m])",
+          "expr": "increase(java_lang_G1_Old_Generation_CollectionTime[10m])",
```
Comment (Wil312) [inline Overview.json]: This needs `{product="confluence"}`

File: Overview.json [hunk 2]
```diff
@@ -441,18 +455,35 @@
...
+          "legendFormat": "{{gc}} ({{instance}})",
           "refId": "A"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
+          },
+          "exemplar": true,
+          "expr": "increase(java_lang_G1_Young_Generation_CollectionTime[10m])",
```
Comment (Wil312) [inline Overview.json]: This needs {product="confluence"}

File: Overview.json [hunk 3]
```diff
@@ -795,25 +833,41 @@
...
       },
       "targets": [
         {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
+          },
           "exemplar": true,
-          "expr": "increase(jvm_gc_collection_seconds_count{instance=\"localhost:9000\"}[1m])",
+          "expr": "increase(java_lang_G1_Old_Generation_CollectionCount[10m])",
```
Comment (Wil312) [inline Overview.json]: This needs {product="confluence"}

File: Overview.json [hunk 4]
```diff
@@ -795,25 +833,41 @@
...
+          "legendFormat": "{{gc}} ({{instance}})",
           "refId": "A"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
+          },
+          "exemplar": true,
+          "expr": "increase(java_lang_G1_Young_Generation_CollectionCount[10m])",
```
Comment (Wil312) [inline Overview.json]: This needs {product="confluence"}

**Review by Wil312 — CHANGES_REQUESTED** [review]
Please see 4 comments left

**Review by Wil312 — APPROVED** [review]

**Pull request merged**